pypdf/_writer.py

```diff
@@ -1291,6 +1391,100 @@ def write(self, stream: Union[Path, StrByteType]) -> Tuple[bool, IO[Any]]:
 
         return my_file, stream
 
+    def list_objects_in_increment(self) -> List[IndirectObject]:
```

Comment:
If this is only for debugging, does this really need to be public?

Reply:
originally I made private, but I've found it useful as it ease to know what will be appended.